src/Text/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { ElementPropTypes } from '@volusion/element-proptypes';
+import { ComponentProps } from '../../types';
+
+interface TextProps extends ComponentProps {
+    content: string;
+}
+
+const Text: React.FC<TextProps> = ({ content }) => {
+    return <div dangerouslySetInnerHTML={{ __html: content }} />;
```

Comment:
What kind of cleaning/filtering will there be for text content?

Reply:
Prod is actually looking at this exact thing or the admin. I assume we'll use whatever they do.